WalletWasabi/BuyAnything/BuyAnythingManager.cs

```diff
@@ -46,6 +46,11 @@ public BuyAnythingManager(string dataDir, TimeSpan period, BuyAnythingClient cli
 	{
 		Client = client;
 		FilePath = Path.Combine(dataDir, "Conversations", "Conversations.json");
+
+		BuyAnythingDataFolder = Path.Combine(EnvironmentHelpers.GetFullBaseDirectory(), "BuyAnything", "Data");
```

Comment:
I am not sure it works, I would need to test it but i think it is not what we want because we have to be sure the path is relative to this assembly and not relative to the executing assembly. Anyway, I can test it tonight.